README.md

```diff
@@ -75,7 +75,7 @@ It can automatically detect mixed numeral formats in a piece of text and convert
 
 ```pycon
 >>> from xnum import convert, NumeralSystem
->>> print(convert("۱۲۳ apples & ٤٥۶ cars", target=NumeralSystem.ENGLISH))
+>>> print(convert("෧෨෩ apples & ꘤꘥꘦ cars", target=NumeralSystem.ENGLISH))
```

Comment:
For the first numbers (`෧෨෩`) please keep it to what it is now; my system can't show it, and I think that'd be the case for most of the users.